src/react_native/safe_area_context.cljs

```diff
@@ -0,0 +1,17 @@
+(ns react-native.safe-area-context
+  (:require ["react-native-safe-area-context"
```

Comment:
Brought this new library to properly report safe areas on Android with  the edge-to-edge mode

Reply:
Also it contains a hook for the safe area (dynamic) and a built-in component that applies safe areas natively

Reply:
Yep, actually we are not using the latest version, the version we are using still supports 0.73. I checked the newer implementation and we can handle it without any problems.

Reply:
BTW, in newer versions they improved their docs